services/web-app/components/footer/footer.js

```diff
@@ -1,74 +1,90 @@
+      </g>{' '}
+    </svg>
+  )
+
   return (
-    <div className={styles.container}>
-      <Grid className={styles.grid}>
-        <Column className={styles.column} sm={4} md={2} lg={2}>
-          <Link href="https://www.google.com">
-            <a className={styles.link}>{'Contribute'}</a>
-          </Link>
-          <Link href="https://www.google.com">
-            <a className={styles.link}>{'Privacy'}</a>
-          </Link>
-          <Link href="https://www.google.com">
-            <a className={styles.link}>{'Terms of use'}</a>
-          </Link>
-          <Link href="https://www.google.com">
-            <a className={styles.link}>{'Ibm.com'}</a>
-          </Link>
-        </Column>
-        <Column className={clsx(styles.column, styles.columnBorderTop)} sm={4} md={2} lg={4}>
-          <Link href="https://www.google.com">
-            <a className={styles.link}>{'Medium'}</a>
-          </Link>
-          <Link href="https://www.google.com">
-            <a className={styles.link}>{'Twitter'}</a>
-          </Link>
-        </Column>
-        <Column
-          className={clsx(styles.column, styles.columnBorderTop)}
-          sm={4}
-          md={3}
-          lg={4}
-          xlg={3}
-        >
-          <p className={styles.text}>
-            {'Have questions? '}
-            <Link href="https://www.google.com">
-              <a className={clsx(styles.link, styles.linkUnderline)}>{'Email'}</a>
-            </Link>
-            {' us for site feedback or open an issue in '}
-            <Link href="https://www.google.com">
-              <a className={clsx(styles.link, styles.linkUnderline)}>{'Github.'}</a>
-            </Link>
-          </p>
-        </Column>
-      </Grid>
-      <Grid>
-        <Column sm={4} md={6} lg={6} xlg={6}>
-          <IbmCloud className={styles.icon} size={64} />
-        </Column>
-      </Grid>
-    </div>
+    <Theme theme="g100" className={styles.container}>
+      <footer>
+        <Grid className={styles.footer}>
+          <Column sm={4} md={2} lg={{ span: 2, start: 5 }} className={styles.column}>
+            <ul>{getColList(colOne)}</ul>
+          </Column>
+          <Column sm={4} md={2} lg={4} className={styles.column}>
+            <ul>{getColList(colTwo)}</ul>
+          </Column>
+          <Column sm={4} md={3} lg={3} className={styles.column}>
+            <p className={styles.text}>
+              Last updated {currentMonth} {currentYear}
+              <br />
+              ©2022 IBM Design Program Office
+            </p>
+          </Column>
+          <Column lg={hasSideNav ? { start: 5 } : { start: 1 }} className={styles.logo}>
```

Comment:
```suggestion
          <Column md={{ start: 1 }}  lg={hasSideNav ? { start: 5 } : { start: 1 }} className={styles.logo}>
```
Need to start at 1 at medium or the logo goes off to the right